frontend/src/i18n.ts

```
@@ -1,13 +1,15 @@
 import { createI18n } from 'vue-i18n'
-import { loadLocales } from '../../common/locales/load.js'
-import { Locale } from '../../common/types.js'
+import { Locale, locales } from '../../common/types.js'
 
-const messages = loadLocales(import.meta.env.VITE_I18N_LOCALES_OVERWRITE)
+const emptyMessages = {} as { [key in Locale]: {} }
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)